distro/testdata/BUILD.tpl

```diff
@@ -1,4 +1,4 @@
-load("@not_named_rules_pkg//:pkg.bzl", "pkg_tar")
+load("@not_named_rules_pkg//pkg:pkg.bzl", "pkg_tar")
```

Comment:
Was this missed in #443?

Reply:
Yes. But becaused of the backwrads compatibility, it still worked.
I tighented it up here.

Reply:
Cool, thanks.